src/main/java/fr/skyost/owngarden/worldedit/WorldEditUtils.java

```diff
@@ -0,0 +1,196 @@
+
+import com.sk89q.worldedit.EditSession;
+import com.sk89q.worldedit.MaxChangedBlocksException;
+import com.sk89q.worldedit.WorldEdit;
+import com.sk89q.worldedit.bukkit.BukkitAdapter;
+import com.sk89q.worldedit.bukkit.BukkitWorld;
+import com.sk89q.worldedit.extent.clipboard.Clipboard;
+import com.sk89q.worldedit.extent.clipboard.io.ClipboardFormat;
+import com.sk89q.worldedit.extent.clipboard.io.ClipboardFormats;
+import com.sk89q.worldedit.function.operation.Operation;
+import com.sk89q.worldedit.function.operation.Operations;
+import com.sk89q.worldedit.math.BlockVector3;
+import com.sk89q.worldedit.math.transform.AffineTransform;
+import com.sk89q.worldedit.session.ClipboardHolder;
+import fr.skyost.owngarden.OwnGarden;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.World;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents available WorldEdit operations.
+ */
+public record WorldEditUtils(OwnGarden plugin) implements Utils {
+
+    /**
+     * Tests if each schematic is valid.
+     *
+     * @return An array containing the invalid schematics.
+     */
+    public boolean testSchematics() {
+        boolean pass = false;
+        for (final Map.Entry<Material, List<Path>> en : plugin.treeMap.entrySet()) {
+            final List<Path> list = new ArrayList<>(en.getValue().size());
+            for (final Path end : en.getValue()) {
+                final Path schem = Path.of(plugin.treeFolder().toString() + File.separator + end.toString());
+                try {
+                    loadSchematic(schem);
+                    Bukkit.getConsoleSender().sendMessage("loaded: " + schem.toString());
+                    list.add(schem);
+                } catch (IOException ex) {
+                    plugin.logger.error(Component.text("Couldn't load " + schem.toString(), NamedTextColor.RED));
+                    pass = true;
+                }
+            }
+            en.setValue(list);
+        }
+        return pass;
+    }
+
+    /**
+     * Returns the file associated with the given schematic.
+     *
+     * @param schematic The schematic.
+     *
+     * @return The file.
+     */
+    private File getFile(final String schematic) {
+        return new File(plugin.schematicsDirectory, schematic);
+    }
+
+    /**
+     * Loads a schematic.
+     *
+     * @param path The schematic file (must be in the schematics directory).
+     *
+     * @return The WorldEdit clipboard holder.
+     *
+     * @throws IOException If any I/O exception occurs.
+     */
+
+    private ClipboardHolder loadSchematic(final Path path) throws IOException {
+        if (!Files.exists(path)) {
```

Comment:
The `!` Can be overseen. Extra into variable.
For example:
```java
if (!Files.exists(path)) {
...
}
```
```java
var pathIsNotExist = !Files.exists(path);
if (pathIsNotExist) {
...
}
```

What is better to read as a newbie ?